### DISCUSSION TITLE
JS-SDK readOne deep limit

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#14761] [Archive]
If i read with readOne an item and its relations (example: meal with related feedbacks), is it possible to set the limit of the deep loaded items?

**Comment by br41nslug** [comment]
You can, https://docs.directus.io/reference/sdk/#read-single-item
```js
await articles.readOne(15, {
	fields: ['*', 'relation.*'],
        deep: { relation: { _limit: 10 } },
});
```

**Reply by NilsBaumgartner1994** [reply]
Nice, but its not documented there

<img width="884" alt="Bildschirmfoto 2022-08-07 um 15 57 57" src="https://user-images.githubusercontent.com/33295329/183294340-57d605cb-2b86-4224-a57c-ff4e309da4f2.png">

**Reply by rijkvanzanten** [reply]
The query in the SDK is the same as https://docs.directus.io/reference/query/ 👍🏻
